SWLOR.Game.Server/Feature/QuestDefinition/TatooineQuestDefinition.cs

```diff
@@ -43,5 +44,42 @@ private static void StinkyWomprats(QuestBuilder builder)
                 .AddGoldReward(1000)
                 .AddXPReward(1750);
         }
+
+        private static void TuskenRampage(QuestBuilder builder)
+        {
+            builder.Create("tusken_rampage", "Tusken Rampage")
+
+                .AddState()
+                .SetStateJournalText("The Militia wants you to kill Fifty Tusken Raider's.")
```

Comment:
This is another one that's just going to piss off the players. Rethink this one too. If you want to make the target a high number that's fine, but don't make them come back for multiple stages.

Reply:
Also fix the grammar please. Run it through ChatGPT or something if you need to.

Reply:
Dropped this to a single stage of hundred and fifty tusken